client/src/Booking.jsx

```diff
@@ -40,56 +32,71 @@ class Booking extends React.Component {
     };
 
     this.init= this.init.bind(this);
+    this.handleClick = this.handleClick.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
     this.book = this.book.bind(this);
     this.bookingTotal = this.bookingTotal.bind(this);
+
   }
 
   componentDidMount() {
     this.init();
-    this.bookingTotal(); //just right now for testing endpt 3
+    this.book();
+    this.bookingTotal(); //testing
   }
 
   init() {
     axios.get('http://localhost:3002/booking')
-    .then((res) => {
-      let site = res.data;
-      let instant = false;
-      if (site.instant_book) {
-        instant = true;
-      }
+    .then(({data}) => {
+      console.log(data)
+      const {
+        name,
+        campId,
+        price_per_night,
+        how_far_out,
+        weeknight_discount,
+        instant_book,
+        cleaning_fee,
+        max_guests,
+      } = data;
       this.setState({
-        name: site.name,
-        campId: site.campId,
-        price_per_night: site.price_per_night,
-        how_far_out: site.how_far_out,
-        weeknight_discount: site.weeknight_discount,
-        instant_book: instant,
-        cleaning_fee: site.cleaning_fee,
-        max_guests: site.max_guests,
+        name,
+        campId,
+        price_per_night,
+        how_far_out,
+        weeknight_discount,
+        instant_book,
+        cleaning_fee,
+        max_guests,
       });
     })
     .catch((err) => {
       throw err;
     });
   }
 
+  handleClick(e) {
+    console.log('e: ', e);
+    e.preventDefault();
+  }
+
   handleSubmit(e) {
     console.log('e: ', e);
     e.preventDefault();
-    this.bookingTotal(e);
+    this.book(e);
   }
 
   //invoked when user clicks checkin button
   book() {
     axios.get('http://localhost:3002/booking/book', { params: { campId: 0 } })
-    .then((res) => {
-      let site = res.data;
-      let month = site.month;
-      let inventory = site.inventory;
+    .then(({data}) => {
+      const {
+        current_month,
```

Comment:
if its just two properties, then id destructure them right in line 92 straight away

Reply:
done ty